### DISCUSSION TITLE
HTTP 503 after installing Docker version

### DISCUSSION THREAD

**Discussion by TimothyG** [Budibase/budibase#4301] [Help]
I've been trying to install Budibase on Docker on a Linux machine. I was able to get everything installed (after changing the platform to linux/amd64 in docker-compose.yaml), but when I go to the URL to run it, I get the following:

HTTP 503
upstream connect error or disconnect/reset before headers. reset reason: connection failure

I'm running on the default port of 10000 and it's on a fresh install of the machine. Is this a known issue?

**Comment by pavleks** [comment]
Hi, @TimothyG I would try removing all your budibase docker containers and then reinstalling them to see if it would change the outcome.
